recipes/deeptime/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "deeptime" %}
+{% set version = "0.0.0" %}
+{% set sha256 = "dd0f3b7df35dab318233f24fb7f96ddf6fd1164504d414814d601c61055e0a07" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - deeptime
```

Comment:
Please include more tests (more import tests are okay)

Reply:
as @cwehmeyer said, this is atm just a placeholder, so there are no more useful tests possible than just importing this.

Reply:
I'm confused, is this code ready to be packaged?

Reply:
Yes, it is. It is in a very early state but that should, as shown below, not be an issue.